assets/js/modules/analytics-4/datastore/audiences.js

```diff
@@ -198,6 +198,17 @@ const baseActions = {
 			return { response: availableAudiences, error };
 		}
 
+		const {
+			response: availableCustomDimensions,
+			error: syncDimensionsError,
+		} = yield commonActions.await(
+			dispatch( MODULES_ANALYTICS_4 ).fetchSyncAvailableCustomDimensions()
+		);
+
+		if ( syncDimensionsError ) {
+			return { response: availableCustomDimensions, error };
+		}
+
```

Comment:
Could you help me understand this change? The IB mentions the available custom dimensions should be synced within the `enableAudienceGroupMain` action. I don't think we need to sync custom dimensions whenever available audiences are synced.